core/src/qos_service.rs

```diff
@@ -244,9 +244,9 @@ impl QosService {
     }
 
     // metrics are reported by bank slot
-    pub fn report_metrics(&self, bank: Arc<Bank>) {
+    pub fn report_metrics(&self, slot: Slot) {
```

Comment:
nit: comment at Ln 18 should be updated as well, or perhaps just remove that comment since it doesn't seem to be necessary

Reply:
done: 9767583d8939642803abb9a1b08cd6979e620015